test/e2e/data/infrastructure-hivelocity/v1alpha1/cluster-template.yaml

```diff
@@ -372,6 +373,7 @@ spec:
           provider-id: DEVICE_ID_PREKUBEADM
           read-only-port: "0"
           resolv-conf: /etc/kubernetes/resolv.conf
+          rotate-server-certificates: "true"
```

Comment:
please move this file into: https://github.com/syself/cluster-api-provider-hetzner/blob/main/.gitignore#L18

Reply:
Done